### DISCUSSION TITLE
HuggingFace type interface

### DISCUSSION THREAD

**Discussion by keithhackbarth** [meltano/meltano#7835] [Ideas]
Random idea here:

I've been using https://huggingface.co/models a lot for data models and also wishing that something similar existed for ETL. 

The Meltano hub is a great resource for me for finding taps however what doesn't exist yet and what I wish did exist is something Hugging Face for ETL. Specifically, if it just cataloged all specifications, tools, taps, and targets available on the internet. The Melanto Hub is close (which is why I'm posting this here) as it does a great job. However, if it was a little more open to other formats, less Meltano-branded, more obviously allowed user contributions, and had a more "tag" style interface - I believe it could be next level! And as a result, would drive more awareness to Meltano itself. 

Meltano is an amazing project, btw!!! Love what you are doing and huge promoter of it!

**Comment by tayloramurphy** [comment]
@keithhackbarth thanks for opening the discussion. 

One of the challenges with your request is that currently everything on the Hub is currently installable and usable with Meltano. The YAML definitions that power the site also power the API which Meltano queries to get the settings, configuration, and details on how a particular plugin works. 

That said, it would be possible to update the API generation to not use specific files. 

My question for you though is what do you mean when you say "all specifications, tools, taps, and targets available on the internet? Would that be something like https://www.moderndatastack.xyz/categories ?
